config/jobs/kubernetes/sig-k8s-infra/trusted/sig-k8s-infra-registry.yaml

```diff
@@ -34,7 +34,7 @@ periodics:
             - name: AWS_WEB_IDENTITY_TOKEN_FILE
               value: /var/run/secrets/aws-iam-token/serviceaccount/token
             - name: AWS_REGION
-              value: us-east-1
+              value: us-east-2
```

Comment:
the actual target bucket is in us-east-2, we want to use the same bucket we were using before

the GCS-to-S3 job is setting this to us-east-1, then overriding it.

the golang AR to S3 tool is not overriding it (we could ...), this is the smallest change for now.